README.md

```diff
@@ -68,117 +99,85 @@ nslookup ${He_Name}.azurecr.io
-# you only need to remember the name of the env file
-source ~/{yoursameuniquename}.env
+# at any point if your terminal environment gets cleared, you can source the file to reload the environment variables and pickup where you left off
+source ~/.helium.env
 
 ```
 
-Create and load sample data into Cosmos DB
+Create Azure Key Vault
 
-- This takes several minutes to run
-- This reference app is designed to use a simple dataset from IMDb of 1300 movies and their associated actors and genres
-- Follow the guidance in the [IMDb Repo](https://github.com/retaildevcrews/imdb) to create a Cosmos DB server (SQL API), a database, and a collection and then load the IMDb data. The repo readme also provides an explanation of the data model design decisions.
-- Recommendation is to set $Imdb_Name the same value as $He_Name
+- All secrets are stored in Azure Key Vault for security
+  - The app uses Managed Identity to access Key Vault in production
+  - and Azure CLI credentials in development
 
 ```bash
 
-# Run saveenv.sh to save the Imdb variables
-./saveenv.sh
-
-# Verify the output of the script shows the newly created Imdb_* variables
+## create the Key Vault
+az keyvault create -g $He_App_RG -n $He_Name
 
 ```
 
-Get the Cosmos DB read only key used by App Service
-
-```bash
-
-export Imdb_Cosmos_RO_Key=$(az cosmosdb keys list -n $Imdb_Name -g $Imdb_RG --query primaryMasterKey -o tsv)
-
-```
+Create and load sample data into Cosmos DB
 
-Create Azure Key Vault
+- This takes several minutes to run
+- This reference app is designed to use a simple dataset from IMDb of 1300 movies and their associated actors and genres
+- Follow the steps in the [IMDb Repo](https://github.com/retaildevcrews/imdb) to create a Cosmos DB server, database, and collection and load the sampleIMDb data.
+  - The repo readme also provides an explanation of the data model design decisions.
+  - Recommendation is to set $Imdb_Name to the same value as $He_Name
+  - `export Imdb_Name=$He_Name`
 
-- All secrets are stored in Azure Key Vault for security
-  - This app uses Managed Identity to access Key Vault
+Save the Cosmos DB config to Key Vault
 
 ```bash
 
-## create the Key Vault and add secrets
-az keyvault create -g $He_App_RG -n $He_Name
-
-# add Cosmos DB keys
+# add Cosmos DB config to Key Vault
 az keyvault secret set -o table --vault-name $He_Name --name "CosmosUrl" --value https://${Imdb_Name}.documents.azure.com:443/
-az keyvault secret set -o table --vault-name $He_Name --name "CosmosKey" --value $Imdb_Cosmos_RO_Key
+az keyvault secret set -o table --vault-name $He_Name --name "CosmosKey" --value $(az cosmosdb keys list -n $Imdb_Name -g $Imdb_RG --query primaryReadonlyMasterKey -o tsv)
+az keyvault secret set -o table --vault-name $He_Name --name "CosmosRWKey" --value $(az cosmosdb keys list -n $Imdb_Name -g $Imdb_RG --query primaryMasterKey -o tsv)
 az keyvault secret set -o table --vault-name $He_Name --name "CosmosDatabase" --value $Imdb_DB
```

Comment:
Since at this point we have already populated the database using the RW key, do we even need to store it in key vault?

Reply:
removed